tests/test_dagmc_file.py

```diff
@@ -78,6 +78,49 @@ def test_set_entityset_ranges():
     assert(all(test_pass))
 
 
+def test_get_meshset_by_id():
+    """
+    Tests the get_meshset_by_id function given valid dim and id
+    """
+    three_vols = ds.DagmcStats(test_env[0]['input_file'])
+    # get volume 0
+    exp = three_vols._my_moab_core.get_entities_by_type_and_tag(
+        three_vols.root_set, types.MBENTITYSET, three_vols.dagmc_tags['geom_dim'], [3])[0]
+    obs = three_vols.get_meshset_by_id('Volumes', ids=[1])
+    assert(obs == exp)
+
+
+def test_get_meshset_by_id_empty_id():
+    """
+    Tests the get_meshset_by_id function given no id
+    """
+    three_vols = ds.DagmcStats(test_env[0]['input_file'])
+    exp = three_vols._my_moab_core.get_entities_by_type_and_tag(
+        three_vols.root_set, types.MBENTITYSET, three_vols.dagmc_tags['geom_dim'], [3])
+    obs = three_vols.get_meshset_by_id('Volumes', ids=[])
+    assert(obs == exp)
```

Comment:
can you add a comment here about what the expected behavior should be if `ids=[ ]` (ie, uses rootset instead)